v4/source/posts.yaml

```diff
@@ -397,6 +397,33 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+  '/channels/{channel_id}/posts/unread':
+    get:
+      tags:
+        - posts
+      summary: Get posts for a channel around last unread
+      description: |
+        Get posts in a channel around last unread post of a member.
+        ##### Permissions
+        Must have `read_channel` permission for the channel.
+      parameters:
+        - name: channel_id
```

Comment:
Is there a parameter to tell how many posts before/after to load?

Reply:
None at the moment.  It's using the default hard-coded value of 60 posts before (including the last unread) and 60 posts after.  I initially thought it's not necessary since we're not doing page request for this API.  However, I realized it makes sense to have.  Should I add something similar to "per_page", maybe like "limit"?

Reply:
I think limit works better than per_page here. We could also make it limit_before and limit_after if we  wanted

Reply:
Submitted PR to mattermost-server adding query of limit_before and limit_after.  Also updated this PR.

Reply:
Is there any reason why we might want to page through this endpoint? Or does that not make sense for the purpose of this endpoint?

Reply:
For our webapp and mobile, maybe we're good at using the default values of 60 at the moment.  Those query parameters will give extra flexibility to other integrations.  0/5 with that though.